dhis-2/pom.xml

```diff
@@ -452,48 +454,6 @@
             </build>
         </profile>
 
-        <profile>
-            <id>dev</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration>
-                            <skipTests>true</skipTests>
-                            <trimStackTrace>false</trimStackTrace>
-                            <argLine>${surefireArgLine}</argLine>
-                            <excludedGroups>integration</excludedGroups>
-                        </configuration>
-                        <dependencies>
-                            <dependency>
-                                <groupId>org.ow2.asm</groupId>
-                                <artifactId>asm</artifactId>
-                                <version>${ow2.asm.version}</version>
-                            </dependency>
-                        </dependencies>
-                    </plugin>
-
-                    <plugin>
-                        <groupId>org.zeroturnaround</groupId>
-                        <artifactId>jrebel-maven-plugin</artifactId>
-                        <version>${jrebel-x-maven-plugin.version}</version>
```

Comment:
can we also remove the plugin version var? not sure if this plugin is still needed

Reply:
Reverting